src/llama_stack_provider_ragas/provider.py

```diff
@@ -1,18 +1,5 @@
-from llama_stack.providers.datatypes import (
-    AdapterSpec,
-    Api,
-    ProviderSpec,
-    remote_provider_spec,
-)
+# remote is the default provider
+from .remote import get_adapter_impl
+from .remote.provider import get_provider_spec
 
-
-def get_provider_spec() -> ProviderSpec:
```

Comment:
this makes sense. But can I ask what a build yaml looks like or this provider?

does module point to `module: llama_stack_provider_ragas` and then this defaults to using the remote `get_provider_spec`?